custom_components/abbfreeathome_ci/translations/en.json

```diff
@@ -88,12 +96,30 @@
           }
         }
       },
+      "dimming_sensor": {
+        "name": "Dimming Event ({channel_id})",
+        "state_attributes": {
+          "event_type": {
+            "state": {
+              "longpress_down": "Dimming Down",
+              "longpress_down_release": "Dimming Down Release",
+              "longpress_up": "Dimming Up",
+              "longpress_up_release": "Dimming Up Release",
+              "off": "Off",
+              "on": "On",
```

Comment:
Are the `on` and `off` translations needed? As they are already referenced to the common translations

Reply:
> Yea, they are. Without this then the On/Off event's won't be translated.

But then I don't get why referencing in strings.json the common-translations `"off": "[%key:common::state::off%]"`. My understanding was, when referncing the common-translations we don't need to care about them and don't need to do double work. Or have I completely misunderstood that?

Reply:
With custom integrations the `strings.json` file isn't used at all, just the language specific files. If this is submitted as a core integration the opposite will be true, you'd only provide the `strings.json` file and no specific languages files.

When I run the integration in my dev environment it automatically generates the language files, which is what I place into this repo to be committed and used.

To be ready to be submitted as a core integration we need to maintain both in this repo.